adapters/AdColony/Public/Headers/GADMAdapterAdColonyExtras.h

```diff
@@ -4,7 +4,7 @@
 
 @import GoogleMobileAds;
 
-@interface GADMAdapterAdColonyExtras : NSObject<GADAdNetworkExtras>
+@interface GADMAdapterAdColonyExtras : GADExtras
```

Comment:
Please use GADNetworkExtras to pass the network specific extras. Using additionalParameters is not preferable.

Reply:
`additionalParameters` is where the `npa` value is stored. If there's another way to access that value I'd gladly use it. Is there another way?

Reply:
Google's documentation says it will be in the `additionalParameters` so no matter what I have to access it through that. https://developers.google.com/admob/ios/eu-consent#forward_consent_to_the_google_mobile_ads_sdk

I can make a property for it, rather than accessing it directly.

Reply:
The documentation you provided is for publishers who are using Google Mobile Ads SDK only. For mediation we need to pass the required consent values through adapter specific extras class, which here is GADMAdapterAdColonyExtras class. Please create a static method in GADMAdapterAdColonyExtras to pass the consent values by the publisher.